Resources/public/js/views/ez-contenteditview.js

```diff
@@ -53,11 +53,15 @@ YUI.add('ez-contenteditview', function (Y) {
                     this._setFocus();
                 }
             });
-            this.on('languageCodeChange', function (e) {
+            this.after('languageCodeChange', function (e) {
+                this.get('formView').set('languageCode', this.get('languageCode'));
```

Comment:
As I have discussed it with @dpobel , we are going to leave *Change events as doing this with setter will lead to situation when calling `this.get('formView')` inside of setter method is not possible while at the time doing that, the `this.get('languageCode')` will be called and for that moment it will return `undefined` or not actual value. Rebuilding `formView` must be done AFTER the attr taking part in it will be set.